src/client.js

```diff
@@ -49,7 +49,8 @@ class Client extends GeneratedApiClient {
     this.apiToken = parsedConfig.client.token;
     this.http = new Http({
       cacheStore: clientConfig.cacheStore,
-      cacheMiddleware: clientConfig.cacheMiddleware
+      cacheMiddleware: clientConfig.cacheMiddleware,
+      fetch: clientConfig.fetch
```

Comment:
I wonder if it'd be nicer to call this something like "requestHandler", "request", "requestExecutor" (java...). Mostly interested if there's some common convention across other libraries. Then again, fetch is the new XHR, so maybe it's better (although very JS specific).

Reply:
I'm debating this too.  I really like request executor.  But I also like that fetch has a defined interface, so I'm not sure it makes sense to re-label the fetch interface as a request executor.